### ISSUE TITLE
Overfitting 막기: Regularization

### ISSUE THREAD

**Issue by sghong977** [sghong977/AIML_Basics#7]
관련자료
- 딥러닝의 일반화와 최적화. 여기 블로그에도 AIML 기본기 관련글이 많다. [링크](https://glanceyes.tistory.com/entry/Deep-Learning-%EC%B5%9C%EC%A0%81%ED%99%94Optimization)

## Regularization.
- [Regularization Normalization과 Regularization](https://velog.io/@sp1rit/)
- [L1, L2 설명](https://huidea.tistory.com/154.)
- L2 Regularization. Weight Decay.
[링크](https://sanghyu.tistory.com/88)
- 일반적인 L1, L2 외에 Regularizer라고 불리는 것들 [링크](https://rain-bow.tistory.com/entry/Semi-Supervised-Learning-SSL-%EC%86%8C%EA%B0%9C-%EB%B0%8F-%EB%8F%99%ED%96%A5)

### Gradient Vanishing 문제
- https://wikidocs.net/61375

### 아무튼 overfitting 막기, Generalization 잘되는 방법은.. 
- regularization. Weight decay 많이 씀
- dropout
- 데이터 다양화를 위해 data augmentation
- 기타 등등
